[Plugin] Global Search Modal

CharrafiMed opened the pull request:
Plugin for Providing an Algolia-Like Search Modal with Highlighting, Favorites, and Recent Searches and more ...

saade reviewed (changes requested):
Hey! Your plugin image must fit the 16:9 aspect ratio, at least 2560x1440 pixels in size, and preferably a JPEG, as per our [Contributing Guide](https://github.com/filamentphp/filamentphp.com#submitting-a-plugin).

Please let us know when you've addressed these issues.

CharrafiMed commented:
> Hey! Your plugin image must fit the 16:9 aspect ratio, at least 2560x1440 pixels in size, and preferably a JPEG, as per our [Contributing Guide](https://github.com/filamentphp/filamentphp.com#submitting-a-plugin).
> 
> Please let us know when you've addressed these issues.
Hey! Sorry it was just an oversight on my own

YoussefACHCHIRAJ commented:
a great and useful plugin, thanks CharrafiMed for this open source plugin, it will very helpful in my future projects
